js/main.js

```diff
@@ -1,9 +1,11 @@
 let display;
+const today = document.getElementById('today');
 
 window.onload = () => {
   // eslint-disable-next-line no-undef
```

Comment:
- Hey, please, let's kindly remove this eslint disable and fix the linter error as the linter helps in writing clean maintainable code to improve our development skill.

Reply:
Hi @zieeco 🖐

I am using classes from various files and used that line that disables the linter since in the requirements it is acceptable.

> Project requirements
![imagen](https://user-images.githubusercontent.com/90431831/221031098-f5d448ee-bac2-4cb9-a70a-ec10ba11a5c4.png)

> Correction

I have changed the name of the classes so that VSCode paints it and disguises that it is a class imported from another js file.
If you can help me with a guide on how to import a class from another file without disabling the linter and not give me an error, I would be very grateful.

| Delete line| Not delete line|
|:------------:|:-----------:|
|![imagen](https://user-images.githubusercontent.com/90431831/221033273-207bc7c7-4c7e-4646-897d-60c80d2ab8da.png)|![imagen](https://user-images.githubusercontent.com/90431831/221034299-e5bd54b1-51e0-4f0e-b0d4-fc9ad1b86b1a.png)|

> Original code
![imagen](https://user-images.githubusercontent.com/90431831/221033793-e6dbfb9d-8242-46ff-bd99-9f46242f8978.png)